### PR TITLE
Elements wiki configuration

### PR DESCRIPTION
**Pull request by DeltaQuad-DQ**

### CODE REVIEW

File: LocalSettings.php [hunk 1]
```diff
@@ -2853,7 +2853,13 @@
 				'confirmed',
 				'autopatrolled',
 				'rollbacker',
-				'tagmanager',
+				'researcher',
+			),
```
Comment (Reception123) [inline LocalSettings.php]: an array is missing here.

File: LocalSettings.php [hunk 2]
```diff
@@ -2853,7 +2853,13 @@
...
+			'bureaucrat' => array(
+				'autopatrolled'
```
Comment (Reception123) [inline LocalSettings.php]: you're missing all commas at the end here

File: LocalSettings.php [hunk 3]
```diff
@@ -3477,7 +3488,13 @@
...
```
Comment (Reception123) [inline LocalSettings.php]: missing commas

**Comment by DeltaQuad-DQ** [comment]
Fixed

**Comment by DeltaQuad-DQ** [comment]
@Reception123 the bureaucrat permission settings don't seem to have gone through

**Comment by Reception123** [comment]
@DeltaQuad-DQ I see. I will ask another sysadmin.

**Comment by Reception123** [comment]
@DeltaQuad-DQ The cause is https://github.com/miraheze/mw-config/blob/master/LocalWiki.php#L70 and https://github.com/miraheze/mw-config/blob/master/LocalWiki.php#L71

**Pull request merged**